src/main/java/gil/server/ParametersMiddleware.java

```diff
@@ -0,0 +1,22 @@
+package gil.server;
+
+import java.io.UnsupportedEncodingException;
+import java.util.function.BiFunction;
+
+public class ParametersMiddleware {
+    public static BiFunction<Request, Response, Response> use =
+            (request, response) -> {
+                String currentResponseBody = response.getBody() == null ? "" : response.getBody();
+                String responseParameters = request.getParameters().toString();
+                Boolean requestHasParameters = responseParameters.length() > 2;
```

Comment:
Similar comment here: what's the significance of the number 2? You can eliminate ambiguity by placing this into a clearly named variable.